website/templates/log_list.mako

```diff
@@ -46,12 +46,17 @@
                                 <span class="contributor-anonymous">A user</span>
                             </span>
                                 <span data-bind="ifnot:log.anonymous">
+                                    <span data-bind="if: log.user.registered">
```

Comment:
It is taken care of with the API. :) @sloria Removed users return an error and it is caught in `logTextParser.js` so it only displays the name.